utilities/utils.py

```diff
@@ -98,8 +100,10 @@ def train_validate_test_normal(
     device = "cpu"
     if torch.cuda.is_available():
         device = "cuda:0"
+        """
```

Comment:
Did the code work before if using the number of GPU available?

Reply:
device = "cuda" also works if the number of visible CUDA devices is bigger than 1. However, it is not clear to me whether one CUDA devices is randomly picked among the ones available, or if by default "cuda:0" is the one always picked. 

"cuda:0" is the local number of GPUs, meaning that if
export CUDA_VISIBLE = 2,3,4
GPU with index = 2 will be "cuda:0"

Reply:
It's probably not crucial since the plan is to entirely switch to DDP, but should we keep the commented DP or just remove?

Reply:
I would keep it just as a placeholder for now.